source/globalCommands.py

```diff
@@ -327,7 +326,7 @@ def script_toggleReportFontName(self,gesture):
 			config.conf["documentFormatting"]["reportFontName"]=True
 		ui.message(state)
 	# Translators: Input help mode message for toggle report font name command.
-	script_toggleReportFontName.__doc__=_("Toggles on and off the reporting of font changes")
+	script_toggleReportFontName.__doc__=_("Toggles on and off the reporting of font name changes")

```

Comment:
As you are touching code for this script, could you please also add the script decorator to this one?